web/src/message_util.ts

```diff
@@ -101,6 +101,27 @@ export function get_messages_in_dm_conversations(user_ids_strings: Set<string>):
         .filter((x) => x.type === "private" && user_ids_strings.has(x.to_user_ids));
 }
 
+export function get_messages_in_topic_from_here(
+    stream_id: number,
+    topic: string,
+    messaged_id: number,
+): Message[] {
+    const all_messages = get_messages_in_topic(stream_id, topic);
+    const messages = [];
+
+    let has_found_the_current_message = false;
+    for (const msg of all_messages) {
+        if (msg.id === messaged_id) {
+            has_found_the_current_message = true;
+        }
+
+        if (has_found_the_current_message) {
+            messages.push(msg);
+        }
```

Comment:
What about this name - `get_current_and_following_messages_in_topic`?

Reply:
Is it okay if I leave the function `get_messages_in_topic_sent_after_current_message`  to return an array of messages instead of the length? I feel it would be consistent with the surrounding code.